packages/react-cosmos-voyager/src/__tests__/index.js

```diff
@@ -1,8 +1,9 @@
 import path from 'path';
+import slash from 'slash';
 import traverse from 'traverse';
 import getFilePaths from '../index';
 
-const resolvePath = relPath => path.join(__dirname, '../use-cases', relPath);
+const resolvePath = relPath => slash(path.join(__dirname, '../use-cases', relPath));
```

Comment:
Unfortunately what ends up happening is that `path.join` will try to use OS specific slashes so it will use backslashes for Windows but we're testing against paths that were processed via glob which [insists on forward slashes](https://github.com/isaacs/node-glob#windows). That's why I'm running everything through `slash`